### PR TITLE
Bug 1220528 - Configure FIREFOX_SYNC=1 for TV only.

### PR DESCRIPTION
**Pull request by weilonge**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bc5e4fd05acbaaee62a72178e0a61cf9fa07fabe

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a2bca60dfaef3a8e58d9cb9a289adb2bab480476

File: Makefile [hunk 1]
```diff
@@ -88,6 +88,10 @@ GAIA_DEV_PIXELS_PER_PX?=1
 NGA_SERVICE_WORKERS?=0
 FIREFOX_SYNC?=0
 
+ifeq ($(GAIA_DEVICE_TYPE),tv)
+  FIREFOX_SYNC=1
+endif
+
```
Comment (ferjm) [inline Makefile]: I'd prefer to add a `tv.mk` (or such) to `build/config/tv` and include it from the global `Makefile`

Comment (weilonge) [inline Makefile]: Great idea!
How about name it as ```device.mk```?
So any devices could place the same file for its build/config folder.
e.g.
```
build/config/tv/device.mk
build/config/phone/device.mk
build/config/tablet/device.mk
```

Comment (ferjm) [inline Makefile]: Sounds good!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f899a461154ec2a2d9f4fb42d6a036d7d8f91804

**Pull request merged**